action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Azure SDK Action'
+description: 'Handle github event'
+inputs:
+  payload:
+    description: 'GitHub event payload'
+    required: true
+  token:
+    description: 'Github event token'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "Handle webhook event..."
+        cd $GITHUB_ACTION_PATH/src
+        echo "$GITHUB_PAYLOAD" > ./github_payload.json
+        go run . ./github_payload.json
+      shell: bash
```

Comment:
Do we expect this to work on windows and mac as well?

Reply:
No, I expect the actions themselves to only run on linux agents, so I'm not worried about the shell type here given it should never be more than few lines (and chose bash for fastest startup). The binary itself I expect to be cross platform though.

Reply:
I suspect bash will work on windows and mac as well we should be careful about making sure our actions don't end up only working on linux even if that is the primary OS.

Reply:
What benefit do we get for supporting cross platform actions in the scope of our event handler type actions? I can see cross platform as necessary for our normal test pipelines, but I don't think those belong in this repository.

For local testing with act, it's all containerized so it shouldn't be an issue.